build_for_ultimaker.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/sh
+#
+# SPDX-License-Identifier: AGPL-3.0+
+#
+# Copyright (C) 2018 Ultimaker B.V.
+# Copyright (C) 2018 Olliver Schinagl <oliver@schinagl.nl>
+#
+
+set -eu
+
+CI_REGISTRY_IMAGE="${CI_REGISTRY_IMAGE:-registry.gitlab.com/ultimaker/embedded/platform/um-kernel}"
+CI_REGISTRY_IMAGE_TAG="${CI_REGISTRY_IMAGE_TAG:-latest}"
+
+WORKDIR="${WORKDIR:-/build}"
+
+trap cleanup EXIT
+
+cleanup() {
+	echo "Cleanup is not yet implemented."
+}
+
+git submodule update --init --recursive
+
+if ! command -V docker; then
+	echo "Docker not found, attempting native build."
+
+	./build.sh "${@}"
+	exit 0
+fi
+
+echo "Starting build using ${CI_REGISTRY_IMAGE}:${CI_REGISTRY_IMAGE_TAG}."
+update_docker_image()
+{
+    if ! docker pull "${CI_REGISTRY_IMAGE}:${CI_REGISTRY_IMAGE_TAG}" 2> /dev/null; then
+        echo "Unable to update docker image '${CI_REGISTRY_IMAGE}:${CI_REGISTRY_IMAGE_TAG}', building locally instead."
+        docker build . -t "${CI_REGISTRY_IMAGE}:${CI_REGISTRY_IMAGE_TAG}"
+    fi
+}
+
+update_docker_image
+
+docker run --rm -i -t -h "$(hostname)" -u "$(id -u)" \
```

Comment:
This build_for_ultimaker file is not up to date with the jedi-system-update one

Reply:
Updated, please review :) .